src/manifest/clog.sidecar

```diff
@@ -24635,4 +24636,4 @@
       }
     ]
   }
-}
+}
```

Comment:
fyi: removed newline

Reply:
Yes. The generated file doesnt have the new line. The old one must have had it because I think I edited it manually.

Reply:
I'll look into making clog add a new line at end of file.